.travis.yml

```diff
@@ -0,0 +1,62 @@
+# Travis configuration file for downloading and building OP-TEE according to the
+# instructions in build.git
+language: bash
+
+notifications:
+  email:
+    recipients:
+      - op-tee@linaro.org
+    on_success: change
+    on_failure: always
+
+dist: trusty
+sudo: required
+group: beta
+
+cache:
+  #ccache: true
```

Comment:
Well, this is one of the magical things with caches, that I've been trying to get working. The thing is that if I do what you suggest, nothing really changed. **But** if I change the `language` a few lines up to `c`, then I can see that Travis picks up the `cache: ccache: true`. But, for some unknown reason I got compiler error in OP-TEE (Python Crypto) for some strange reason. See this for example: https://travis-ci.org/jbech-linaro/build/builds/195668777

So, for now, I'd like to keep `$HOME/.ccache` and instead remove the out-commented `#ccache: true` and then later try to see if it's possible to change to language `c` and resolve the issue mentioned.